course_catalog/constants.py

```diff
@@ -116,11 +116,13 @@ class PrivacyLevel(Enum):
     "Computer Science": ["Computer Science"],
     "Crisis Management": ["Management"],
     "Data Modeling & Analytics": ["Computer Science"],
+    "Digital Transformation": ["Business", "Information Technology"],
```

Comment:
```suggestion
    "Digital Transformation": ["Information Technology"],
```
"Business" is too broad a match for this. I think just "Information Technology" is OK.